test/unit/configmap_spec.js

```diff
@@ -6,14 +6,13 @@ describe('Examine Config Maps',  function () {
 
     let configMaps
     let services
-    let ingresse
     let deployments
     before(async function () {
        const { docs } = await setup()
-       configMaps = docs.filter(d => d.kind === 'ConfigMap')
-       services = docs.filter(d => d.kind === 'Service')
-       ingresses = docs.filter(d => d.kind === 'Ingress')
-       deployments = docs.filter(d => d.kind === 'Deployment')
+       configMaps = docs.filter(d => d && d.kind === 'ConfigMap')
+       services = docs.filter(d => d && d.kind === 'Service')
+       ingresses = docs.filter(d => d && d.kind === 'Ingress')
```

Comment:
```suggestion
```

`services` and `ingresse` appear to be unused in `test/unit/helm_spec.js`

If kept for future use, then it looks like the decleration `let ingresse` has been deleted.